include/circt/Dialect/Seq/SeqOps.td

```diff
@@ -78,13 +78,13 @@ def CompRegClockEnabledOp : SeqOp<"compreg.ce",
 
   let builders = [
     OpBuilder<(ins "Value":$input, "Value":$clk, "Value":$ce,
-                   "StringRef":$name), [{
+                   "Twine":$name), [{
       auto nameAttr = StringAttr::get($_builder.getContext(), name);
       return build($_builder, $_state, input.getType(), input, clk, ce,
                    nameAttr, Value(), Value(), hw::InnerSymAttr::get(nameAttr));
     }]>,
     OpBuilder<(ins "Value":$input, "Value":$clk, "Value":$ce,
-                   "Value":$reset, "Value":$rstValue, "StringRef":$name),
+                   "Value":$reset, "Value":$rstValue, "Twine":$name),
```

Comment:
Please keep gratuitous changes out of this feature pr.

Reply:
removed